charts/nextcloud/templates/nginx-config.yaml

```
@@ -173,10 +144,12 @@ metadata:
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
 data:
-  nginx.conf: |-
 {{- if .Values.nginx.config.default }}
-    {{- template "nginx.conf" $ }}
-{{- else }}
+  default.conf: |-
+    {{- template "default.conf" $ }}
+{{- end }}
+{{- if .Values.nginx.config.custom }}
```

indent golang like yaml + use with instatt of if

```suggestion
  {{- with .Values.nginx.config.custom }}
  zz-custom.conf: |-
    {{ . | nindent 4 }j
  {{- end }]
```